mlxtend/evaluate/scoring.py

```diff
@@ -111,10 +120,10 @@ def scoring(y_target, y_predicted, metric='error',
     elif metric == 'error':
         res = _error(targ_tmp, pred_tmp)
     elif metric == 'average per-class accuracy':
```

Comment:
I think there was a misunderstanding. The per-class accuracy would not be used to replace the average-per-class-accuracy. Maybe have a look at the brief comment in #615

Reply:
Yes! This makes much more sense now. Thanks!